### DISCUSSION TITLE
Browsing /graphql to access GraphQL Playground - GET query missing

### DISCUSSION THREAD

**Discussion by rhvitfeldt** [requarks/wiki#5926] [Help / Questions] [answered]
Hi all,

According to https://docs.requarks.io/en/dev/api we should be able to access GraphQL Playground tool by browsing to https://ourwiki/graphql. But this page only shows "GET query missing.". Is this information old? Is the function removed?

OS: Ubuntu 20.04
Wiki.js version: 2.5.277
Database engine: PostgreSQL 12.12

**Comment by NGPixel** [comment, marked as answer]
Probably because you have an environment variable NODE_ENV set to `production`. Either unset or set it to `development` to get access to the graphql playground.

Otherwise you can use a tool like Insomnia or Postman to build queries / see schema documentation.

**Reply by rhvitfeldt** [reply]
I changed the NODE_ENV variable to development and still had the issue. Didn't reboot the server afterwards.

But I followed this instruction to get it working:
https://github.com/requarks/wiki/discussions/3254#discussioncomment-1042597

**Reply by NGPixel** [reply]
Well you do need to reboot the server...

**Reply by rhvitfeldt** [reply]
> Well you do need to reboot the server...

Roger! Thanks.

**Comment by harvthe2** [comment]
Was helpful to know the NODE_ENV is in the wiki.service you create as part of the installation. 
/etc/systemd/system/wiki.service in my case
If you change the wiki.service you will need to do a systemctl daemon-reload and then restart the service for it to take effect.
